example/cluster-machine-template-simple.yaml

```diff
@@ -95,28 +86,28 @@ spec:
             flow: Inbound
             ipProtocol: tcp
 # IpRange to authorize access to kubernetes endpoints (kube-apiserver), you must keep it and change it with a CIDR that best suits with your environment.
-            ipRange: "10.0.0.128/26"
+            ipRange: "10.0.0.128/24"
             fromPortRange: 10250
             toPortRange: 10250
           - name: cluster-api-securitygrouprule-api-kubelet-kcp
             flow: Inbound
             ipProtocol: tcp
 # IpRange to authorize access to kubernetes endpoints (kube-apiserver), you must keep it and change it with a CIDR that best suits with your environment.
-            ipRange: "10.0.0.32/28"
+            ipRange: "10.0.0.32/24"
```

Comment:
Please changes with the same default values in exemple.

(Here also those /24 are not valid)